pkg/machinepool/machinepool.go

```diff
@@ -29,6 +33,49 @@ func NewMachinePoolService() MachinePoolService {
 	return &machinePool{}
 }
 
+// ListMachinePools lists all machinepools (or, nodepools if hypershift) in a cluster
+func (m *machinePool) ListMachinePools(r *rosa.Runtime, clusterKey string, cluster *cmv1.Cluster, isHypershift bool) {
```

Comment:
Is it possible to add some test coverage for ListMachinePools so that the first half of the code is also tested?

Reply:
Next ticket in this task will be to move the command to a new runner, which will allow easier testing of functions like this which require a runtime -- will be attempting to get 100% coverage in that MR

WDYT?

Reply:
@hunterkepley Sounds good, feel free to resolve the thread - I don't have that permission